app/gui/src/main/scala/org/bitcoins/gui/util/GUIUtil.scala

```diff
@@ -31,6 +38,78 @@ object GUIUtil {
       .format(instant)
   }
 
+  def setStringToClipboard(str: String): Unit = {
+    val clipboard = getDefaultToolkit.getSystemClipboard
+    val sel = new StringSelection(str)
+    clipboard.setContents(sel, sel)
+  }
+
+  // fileChooser persists so initialDirectory can update across interactions
+  private lazy val fileChooser = new FileChooser() {
+    extensionFilters.addAll(txtExtensionFilter, allExtensionFilter)
+    selectedExtensionFilter = txtExtensionFilter
+    initialDirectory = new File(Properties.userHome)
+  }
+
+  private lazy val txtExtensionFilter =
+    new ExtensionFilter("Text Files", "*.txt")
+  private lazy val allExtensionFilter = new ExtensionFilter("All Files", "*")
+
+  def showSaveDialog(
+      bytes: String = null,
```

Comment:
get rid of `null`s

Reply:
It's used pretty pervasively in this file, we need to remove all usages and replace with `Option`. 


It seems like we are using default arguments quite a bit too. Generally this should be used sparingly.